package.json

```diff
@@ -4,23 +4,26 @@
     "private": true,
     "scripts": {
         "start": "webpack-dev-server --port=4200 --host 0.0.0.0 --open",
-        "clean": "del ./dist",
-        "cpy": "cpy",
+        "clean": "shx rm -rf ./dist",
         "build": "npm-run-all clean build:angular webpack build:icons",
         "pretest": "webpack",
         "test": "karma start ./karma.conf.js",
         "test:travis": "npm-run-all clang:check tslint:check test",
         "clang:check": "./scripts/clang-check.sh",
         "clang:format": "node_modules/.bin/clang-format -style=file -i --glob=src/**/*.ts",
-        "prebuild:angular": "cpy ./npm/clarity-angular/package.json ./src/clarity-angular/; replace '@VERSION' $npm_package_version ./src/clarity-angular/package.json",
+        "prebuild:cp": "shx cp ./npm/clarity-angular/package.json ./src/clarity-angular/",
+        "prebuild:ver": "cross-conf-env replace-in-file @VERSION npm_package_version ./src/clarity-angular/package.json",
+        "prebuild:angular": "npm-run-all prebuild:cp prebuild:ver prebuild:ver",
         "build:angular": "ng-packagr -p ng-package.json",
-        "postbuild:angular": "cpy dist/clarity-angular/bundles/* dist/clarity-angular; cd dist/clarity-angular; npm pack;",
+        "postbuild:cp": "shx cp dist/clarity-angular/bundles/* dist/clarity-angular",
+        "postbuild:cd": "cd dist/clarity-angular",
+        "postbuild:pack": "npm pack",
+        "postbuild:bundle": "npm-run-all postbuild:cp postbuild:cd postbuild:pack",
```

Comment:
I would prefer to keep the previous pattern of

``` 
"postbuild:angular": "npm-run-all shx cp dist/clarity-angular/bundles/* dist/clarity-angular cd dist/clarity-angular npm pack" 
```

What prompted breaking them out into individual script commands? 
@jeeyun - what do you think?